src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -84,40 +84,169 @@ def get(self, svc_id):
         return daemon
 
 
-@ApiController('/rgw/proxy')
-@AuthRequired()
-class RgwProxy(BaseController):
+class RgwRESTController(RESTController):
```

Comment:
This class does not use anything from the `RESTController` class, and the `proxy` method doesn't also use the `self` variable. I would remove this `RgwRESTController` and put the `proxy` method as a module function that is accessible to all controllers declared in this file.

Reply:
I would like to leave it as it is because the ``RgwRESTController```class is a specialisation of the ``RESTController`` class especially for the RGW needs. I'm right with you, it does not use anything from the derived class, but it's a logical layer that simplifies the implementation using OOP.

Reply:
OK :-)